pkg/controller/servicebinding/reconciler.go

```diff
@@ -198,7 +198,6 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	if err != nil {
 		return requeueError(err)
 	}
-
 	options := &serviceBinderOptions{
```

Comment:
Revert this change, it seems the only one in this PR and is unrelated to the changelist.